tox.ini

```
@@ -108,7 +108,7 @@ commands =
     !cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} {posargs}
     cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} --cov astropy --cov-config={toxinidir}/pyproject.toml --cov-report xml:{toxinidir}/coverage.xml {posargs}
 
-    double: python -c 'import sys; from astropy import test; test(); sys.exit(test())'
+    double: pytest --keep-duplicates --pyargs astropy astropy
```

Do I have to do something like this?

```suggestion
    double: pytest --keep-duplicates --pyargs astropy astropy {toxinidir}/docs {toxinidir}/docs {env:MPLFLAGS} {posargs}
```

---

Yes, I think so, and probably do `astropy {doxinidir}/docs astropy {toxinidir}/docs`

---

I think we can skip `{env:MPLFLAGS}` since, if I understand correctly, it'll always be empty in this context.
Also: the *current* implementation for `double` tests doesn't include doctests for `docs/`. We could add it later but that seems out of scope here.

Otherwise: neat ! I had no idea `--keep-duplicates` existed *or* that `--pyargs`
- could be used with more than one package name
- still falls back to interpreting arguments as fs-paths when they don't match an installed package name